middleware/cache/README.md

```diff
@@ -60,3 +60,13 @@ Proxy to Google Public DNS and only cache responses for example.org (or below).
 proxy . 8.8.8.8:53
 cache example.org
 ~~~
+
+### TTL examples  
+no cache -> record TTL is used  
+*cache* -> cached with a TTL of max 3600 or 1800 - for smaller record provided TTLs the smaller one is used  
+*cache 0* -> cached with a TTL of 5  
```

Comment:
These are also not actually examples as the explain what TTL value does. I think this should be moved up in the file.